include/tim/vx/ops/conv2d.h

```diff
@@ -95,6 +95,13 @@ class Conv2d : public BuiltinOp {
   const std::array<uint32_t, 4> pad_;
   const int32_t multiplier_;
   const DataLayout kernel_layout_;
+
+#if defined(__clang__) && (__clang_major__ >= 15)
```

Comment:
// compiler_feature.h so that we can extend the compiler support
#if defined(__clang__) && (...)
#define SUPPORT_FLOAT16
#endif 

#ifdef SUPPORT_FLOAT16
#define TIM_VX_OPS_CONV2D
#endif